governance/ChaosDAOInternalProposals.md

```diff
@@ -0,0 +1,16 @@
+# Internal ChaosDAO Governance Procedures
+
+## How to Propose?
+Generally, proposals often need some initial discussion to hammer specific proposal parameters. In those cases, DAOists should first bring up proposal topics in a new #daist-forum discussion thread. Some quick single-parameter votes can be held, even including multiple options for people to vote between. That provides insight into what exactly the proposal should include.
+
+Once the details of a proposal are hammered out, a DAOist should make a new post in the #proposals forum with only a single up/down vote (like for on-chain referenda).
+
+## How to Vote
+Voters use emojis (usually thumbs up/down) to vote on a proposal.
+* Using a thread within a normal channel, voting is done on the initial post so that the votes are visible before entering the thread. This works because all emojie votes are visible.
+* If posting as a new topic in a Discord "forum", Only the first emoji symbol used will show a total at the top level in the list of forum topics. Therefore for anything other than a simple thumbs-up vote in a forum topic, the voting should happen in one of the first comments in that discussion topic to avoid any bias for/against a particular outcome. 
```

Comment:
Lower case O on the word "only" after the comment

Reply:
👍